etc/profile-m-z/xarchiver.profile

```diff
@@ -0,0 +1,21 @@
+# Firejail profile for xarchiver
+# Description: Lightweight desktop independent archive manager
+# This file is overwritten after every install/update
+# Persistent local customizations
+include xarchiver.local
+# Persistent global definitions
+include globals.local
+
+ignore blacklist ${RUNUSER}
+ignore dbus-user none
+ignore x11
+ignore include disable-shell.inc
+
+noblacklist ${HOME}/.config/xarchiver
+
+noroot
```

Comment:
```suggestion
```

This is commented in archiver-common, probably to avoid issues with group
permissions.